conda/recipes/dask-cudf/meta.yaml

```diff
@@ -41,18 +41,27 @@ requirements:
     - dask >=2023.5.1
     - dask-core >=2023.5.1
     - distributed >=2023.5.1
+    {% if cuda_major == "11" %}
     - cudatoolkit ={{ cuda_version }}
+    {% endif %}
```

Comment:
Do we need `cudatoolkit` here since it is not needed by `dask-cudf` directly and `cudf` would pull it in for us?

```suggestion
```

Reply:
I avoided changing CUDA 11 packaging in this PR, but I agree this might be safe.

Reply:
Makes sense. Let's save it for follow up then

Reply:
https://github.com/rapidsai/cudf/issues/13607#issuecomment-1603333318